graphistry/plugins/kustograph.py

```diff
@@ -2,6 +2,7 @@
 import time
 import json
 from typing import Any, Dict, List, Optional
+from typing_extensions import TypedDict, NotRequired
```

Comment:
What happens if the user does not have `azure.kusto.*` installed, will pygraphistry fail to import?

should we make these conditional somehow ,like

1. Conditional top-level imports

```python
if typing.TYPE_CHECKING:
  try:
    from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
  catch ModuleImporterror:
    KustoConnectionStringBuilder = Any
```

2. Lazy runtime imports

```python
def _connect(self) -> KustoClient:
    # trigger better runtime exn on use
    from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
```

Reply:
fwiw, louie can build & release off of a PR version here, so fixing this is just about landing in pyg main ... not great but allows progress